backend/api/Startup.cs

```diff
@@ -138,6 +141,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IKeycloakRequestClient, KeycloakRequestClient>();
 
             services.AddAutoMapper(typeof(Startup));
+
+            services.AddHealthChecks()
+                .AddCheck("liveliness", () => HealthCheckResult.Healthy())
```

Comment:
this is just a simple function that always returns true.

Reply:
Is this an endpoint `/api/liveliness`?

Reply:
currently just liveliness, but I can add the /api